apps/demo-react/components/wallet-fallback/useErrorMessage.ts

```diff
@@ -0,0 +1,19 @@
+import {
+  useSupportedChains,
+  useConnectorError,
+  helpers,
+} from 'reef-knot/web3-react';
+import { useNetwork } from 'wagmi';
+
+export const useErrorMessage = (): string | undefined => {
+  const error = useConnectorError();
+  const { isUnsupported } = useSupportedChains();
+  const { chains: supportedChains } = useNetwork();
+
+  // TODO: fix useConnectorError in reef-knot and remove this block
```

Comment:
What needs to be fixed?

Reply:
Here used deprecated 'getUnsupportedChainError'